CTFd/api/v1/challenges.py

```diff
@@ -201,6 +200,8 @@ def get(self, query_args):
                     "type": challenge_type.name,
                     "name": challenge.name,
                     "value": challenge.value,
+                    "solved": challenge.id in user_solves,
```

Comment:
This will make it possible to avoid an extra query for solves when colouring the challenges. I won't do that in this changeset but definitely something to think about as follow on work.

Reply:
Yup I will handle that in a follow up PR